src/main/java/com/powsybl/openloadflow/ac/nr/NewtonRaphson.java

```diff
@@ -64,7 +66,36 @@ public static List<Pair<Equation<AcVariableType, AcEquationType>, Double>> findL
                 .collect(Collectors.toList());
     }
 
-    private NewtonRaphsonStatus runIteration(StateVectorScaling svScaling) {
+    public static List<Pair<Equation<AcVariableType, AcEquationType>, Double>> findLargestMismatch(EquationSystem<AcVariableType, AcEquationType> equationSystem, double[] mismatch, AcEquationType equationType) {
```

Comment:
No need for a list, you'd better return an `Optional<Pair<Equation<AcVariableType, AcEquationType>, Double>>`